public/app/features/integrated-alerting/components/Alerts/AlertsTable/AlertsTable.styles.ts

```diff
@@ -0,0 +1,104 @@
+import { css } from 'emotion';
+import { GrafanaTheme } from '@grafana/data';
+import { stylesFactory } from '@grafana/ui';
+
+export const getStyles = stylesFactory((theme: GrafanaTheme) => {
```

Comment:
```suggestion
export const getStyles = (theme: GrafanaTheme) => {
```
No need to use the factory as per https://github.com/grafana/grafana/blob/master/contribute/style-guides/themes.md#usestyles-hook

Reply:
Fixed and fixed in `AlertRulesTable.styles.ts` as well